main/src/ca/uwaterloo/flix/language/ast/AtomicOp.scala

```diff
@@ -98,4 +98,6 @@ object AtomicOp {
 
   case object MatchError extends AtomicOp
 
+  case object Throw extends AtomicOp
```

Comment:
Could we move Throw after PutStaticField? That way its group with other JVM operations.

The same goes for pattern matches etc.